DDD.Core/AzureStorage/SessionFeedbackEntity.cs

```diff
@@ -1,13 +1,29 @@
-﻿using Microsoft.WindowsAzure.Storage.Table;
+﻿using System;
+using Microsoft.WindowsAzure.Storage.Table;
 
 namespace DDD.Core.AzureStorage
 {
     public class SessionFeedbackEntity : TableEntity
     {
+        public SessionFeedbackEntity() {}
+
+        public SessionFeedbackEntity(string sessionId, string conferenceInstance, string name, string rating, string liked, string improvementIdeas, string sessionName) {
+            PartitionKey = conferenceInstance;
+            RowKey = Guid.NewGuid().ToString();
+            SessionId = sessionId;
+            Name = name;
```

Comment:
This can probably be null too if there is a session id?

Reply:
This "name" variable is the attendee name who is giving the feedback.
Don't think we should remove it.